content/blog/configuring-sso-for-hpe-greenlake-central-private-cloud-enterprise-and-hpe-greenlake-glcp-using-okta.md

```diff
@@ -0,0 +1,166 @@
+
+![](/img/saml_settings.jpg)
+
+2﻿. Complete the setup.
+
+![](/img/ws-image7.png)
+
+Click Next and Select “Internal App”, then Finish.
+
+**Step 3:** **Export the SAML 2.0 IdP metadata**
+
+1. Click Next – Configure the Sign On settings
+
+   You will find two options are available: **View Setup Instructions** which steps you through the SAML configuration and **Identity Provider metadata**, which will produce an XML file that can be loaded into HPE GreenLake Cloud Platform (GLCP) application
+
+   Suggestion: Click **Identity Provider metadata** and save the XML data to a file.
+
+   ![](/img/ws-image9.png)
+2. C﻿lick Next.       
+3. Select Internal app, and Click Finish.    
+
+   **Step 3.1 :  Access to the SAML application and HPE Greenlake is determined by assigning only those members or group to the SAML application.**
+
+![](/img/customer-user-assignment-to-saml.jpg)
+
+
+
+**S﻿teps to be followed in HPE GreenLake Cloud Platform(GLCP)**
+
+ **   Step 4: Create SAML Authorization Profile in HPE GreenLake Cloud Platform**
+
+1. Log into **HPE GreenLake Cloud Platform** and click Menu > Manage > Authentication and Click Set Up SAML Connection.
+
+   *Before you can add a new SAML configuration, you must have at least one user account with that domain already enabled in HPE GreenLake. Also, you must be logged into HPE GreenLake with an account from that domain in order to enable SSO for it.*
+
+   ![](/img/ws-image10.png)
+2. Type in the domain you want to enable SSO on:
+
+   ![](/img/glp_domain.jpg)
+3. Input the metadata from the step above.
+
+   While HPE GreenLake does support entering this information manually, it's recommended that you simply upload the XML metadata that was downloaded in the previous step. To do so, Select Metadata File, selecting the XML file. Then, click Next.
+
+   ![](/img/ws-image12.png)
+4. Enter the SAML attributes to match what was entered in Okta. Set the idle timeout value as well.
+
+   ![](/img/config_setting_sso_appjpg.jpg)
+5. Then click Next.    
+6. Create a recover user so that, in the event SSO fails, an admin will still be able to access the HPE GreenLake portal.    
+
+   ![](/img/recovery_user.jpg)
+
+   Congratulations! SSO will now be enabled for HPE GreenLake as well as the HPE GreenLake Cloud Platform (GLCP) application. Log out and on the HPE GreenLake home page, click **Sign in with SSO**.
+
+**Testing and troubleshooting:**
+
+On the HPE GreenLake Cloud Platform home page, click **Sign In with SSO**.
+
+![](/img/ws-image15.png)
+
+![](/img/hpe-greenlake-sso-page.jpg)
+
+Enter the SSO credentials. You will be redirected to Okta to authenticate. Once you successfully authenticate, you will be redirected back to HPE GreenLake. You can then click on the HPE GreenLake Cloud Platform (GLCP) application and be given access based on the configured role/permissions.
+
+**Additional notes:**
+
+* There must be at least **one** verified user belonging to the **Domain** prior to configuration.    
```

Comment:
This section is very nicely formatted. You had another section earlier that said something similar. Would it make sense to format them similarly?